CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+# ***************************************************************************
+# Created:   Mon  3 May 14:08:27 CEST 2021
+#
+# Copyright  2021  Till Hofmann <hofmann@kbsg.rwth-aachen.de>
+# ****************************************************************************/
+# This program is free software; you can redistribute it and/or modify it under
+# the terms of the GNU General Public License as published by the Free Software
+# Foundation; either version 2 of the License, or (at your option) any later
+# version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.  See the GNU Library General Public License for more
+# details.
+#
+# Read the full text in the LICENSE.md file.
+#
+
+cmake_minimum_required(VERSION 3.14)
+
+project(gazebo-rcll)
+include(GNUInstallDirs)
+
+set(USERDIR
+    ".config/rcll-refbox"
```

Comment:
Is there a reason why this is named rcll-refbox?

Reply:
Good catch, it should be `gazebo-rcll` instead. Or even better `.config/rcll/gazebo`?